### ISSUE TITLE
When clicking on attachment icon nothing is happening 

### ISSUE THREAD

**Issue by patelvivek9115** [dimelo/Dimelo-iOS#37]
When I am clicking on attachment icon no action sheet or other option is opening, also after some bottom portion of char screen is black and after closing the view and when I am reopening it the bottom input bar is invisible

**Comment by patelvivek9115** [comment]
When I am clicking on attachment icon no action sheet or other option is opening, also after some bottom portion of char screen is black and after closing the view and when I am reopening it the bottom input bar is invisible

**Comment by waelba** [comment]
Hi @patelvivek9115 ,

Can you provide with these informations please:

Which SDK version are you using ?
Can you give me a snippet of your code ?

Thank you.

**Comment by patelvivek9115** [comment]
I am using the latest one and here is my code sample, also my device is iPhone 11 and iOS version is 13.3.1

```
       let dimelo: Dimelo = Dimelo.sharedInstance()

        guard let controller = dimelo.chatViewController() else{
            return
        }

        controller.modalPresentationStyle = .fullScreen

        let barItem = UIBarButtonItem(image: #imageLiteral(resourceName: "back_white_arrow"), style: .plain, target: self, action: #selector(actionBack))

        barItem.tintColor = .white

        controller.navigationItem.leftBarButtonItem = barItem

        self.tabBarController?.present(controller, animated: true, completion: nil)
```

**Comment by patelvivek9115** [comment]
![IMG_5025](https://user-images.githubusercontent.com/14240280/76847232-c25cc000-6867-11ea-9913-10653877f5e0.PNG)
![IMG_5026 2](https://user-images.githubusercontent.com/14240280/76847256-c852a100-6867-11ea-8c16-2dcd45c8050a.PNG)

**Comment by waelba** [comment]
One more question, you use a **SceneDelegate** in your projet ?

**Comment by patelvivek9115** [comment]
Yes I have used, also I am not calling the code from AppDelegate class, it's my other controller from which I am opening the chat view.

**Comment by waelba** [comment]
OK I'll take a look and I'll come back to you. Thank you

**Comment by patelvivek9115** [comment]
Sure thanks

**Comment by patelvivek9115** [comment]
@wa-el Please let me know if you have found any solution?

**Comment by waelba** [comment]
Hi @patelvivek9115 ,

It's fixed in v2.2.0

Thanks
